autopts/ptsprojects/mynewt/gap.py

```diff
@@ -126,6 +130,8 @@ def set_pixits(ptses):
                   "TSPX_iut_device_name_in_adv_packet_for_random_address", "")
     pts.set_pixit("GAP", "TSPX_Tgap_104", "60000")
     pts.set_pixit("GAP", "TSPX_URI", "000168747470733A2F2F7777772E626C7565746F")
+    pts.set_pixit("GAP", "TSPX_periodic_advertising_data",
+                  binascii.hexlify((str(periodic_data[0]) + str(len(periodic_data[1])) + periodic_data[1]).encode()))
```

Comment:
I found one more issue at  GAP/PADV/PAM/BV-01-C test case:
"periodic advertisement data doesn't match TSPX_periodic_advertising_data"
caused by a little mistake at constructing LTV data (str instead of chr, Type before Length)
```suggestion
    pts.set_pixit("GAP", "TSPX_periodic_advertising_data",
                  binascii.hexlify((chr(len(periodic_data[1]) + 1) + chr(periodic_data[0]) +
                                    periodic_data[1]).encode()))
```
I might bother your a few times more, because currently I'm working at padv support for Zephyr :)

Reply:
No problem :) I adjusted spec too to LTV instead of TLV